src/SAWScript/Proof.hs

```diff
@@ -378,6 +379,13 @@ unfoldProp sc unints (Prop tm) =
   do tm' <- scUnfoldConstantSet sc True unints tm
      return (Prop tm')
 
+-- | Unfold all the constants appearing in the proposition
+--   whose VarIndex is found in the given set.
```

Comment:
These Haddocks appear to be copy-pasted from `unfoldProp`, but they don't mention anything about fixpoints, which is a crucial difference between this function and `unfoldProp`.